### ISSUE TITLE
Create a Hostel Management System

### ISSUE THREAD

**Issue by DarkTech7** [DarkTech7/oss-project#20]
I want a Hostel Management System using any tech stack of your choice, you can discuss with your fellow contributors for the tech stack.

The Project Deadline is 1 November 2024. The following Basic Features are required:
- Student management system 
- Student Safety System 
- Facility Manager 
- Staff Management System 
- Regular Maintenance System 
- Rent Management System 
- Document Management System 
- Electricity and Water Management System 
- Complaint Management System

**Comment by DarkTech7** [comment]
@theavnishkumar @adityai0 @shubhammukherjee17 @yuvrajkourav2457 @DarkTech7 
Kindly work on this project for your hacktoberfest 2024.

**Comment by adityai0** [comment]
Hi @DarkTech7 ,

Thanks for opening the issue. Here's my proposal for the Hostel Management System:

**Tech Stack:**

- **Frontend:** Next.js , Tailwind CSS
- **Backend:** Node.js, MongoDB, Mongoose
- **Auth:** JWT
- **Deployment:** Vercel (frontend), Azure (backend)
- **Other Tools:** Zod, React Hook Form, Git, GitHub

**Core Features:**

1. Student Management
2. Facility Management
3. Staff Management
4. Maintenance System
5. Rent Management
6. Complaint Management

We can start development at the end of August.

Please review and let me know your thoughts.

**Comment by Tara-cloud2917** [comment]
> I want a Hostel Management System using any tech stack of your choice, you can discuss with your fellow contributors for the tech stack.
> 
> The Project Deadline is 1 November 2024. The following Basic Features are required:
> - Student management system 
> - Student Safety System 
> - Facility Manager 
> - Staff Management System 
> - Regular Maintenance System 
> - Rent Management System 
> - Document Management System 
> - Electricity and Water Management System 
> - Complaint Management System 
> 
> 

I also want hostel management system

**Comment by adityai0** [comment]
@Tara-cloud2917 Your Good Name?

**Comment by Tara-cloud2917** [comment]
> @Tara-cloud2917 Your Good Name?

Tara can u provide hms?

**Comment by adityai0** [comment]
Yes we can build HMS. But without proper identification we can't proceed as your account looks spammy.

**Comment by Tara-cloud2917** [comment]
> proceed

Ok but how much you take for creating hms?

**Comment by DarkTech7** [comment]
Provide us your true identity?? @Tara-cloud2917
